ecosystem/sep-0010.md

```diff
@@ -146,38 +146,36 @@ Every other HTTP status code will be considered an error. For example:
+This endpoint accepts a signed challenge transaction, validates it and responds with a session [JSON Web Token](https://jwt.io/) authenticating the account.
 
-Client submits a challenge transaction (that was previously returned by the [`challenge`](#challenge) endpoint) as a HTTP POST request to `WEB_AUTH_ENDPOINT` using one of the following formats (both should be equally supported by the server):
+The **Client Application** submits a challenge transaction (that was previously returned by the [`challenge`](#challenge) endpoint) as a HTTP POST request to `WEB_AUTH_ENDPOINT` using one of the following formats (both should be equally supported by the server):
 
 * Content-Type: `application/x-www-form-urlencoded`, body: `transaction=<signed XDR (URL-encoded)>`)
 * Content-Type: `application/json`, body: `{"transaction": "<signed XDR>"}`
 
-To validate the challenge transaction the following steps are performed by the server. If any of the listed steps fail, then the authentication request must be rejected — that is, treated by the application as an invalid input.
+To validate the challenge transaction the following steps are performed by the **Web Auth Endpoint**. If any of the listed steps fail, then the authentication request must be rejected — that is, treated by the **Web Auth Endpoint** as an invalid input.
 
 * decode the received input as a base64-urlencoded XDR representation of Stellar transaction envelope;
-* verify that transaction source account is equal to the server's signing key;
+* verify that transaction source account is equal to the **Home Domain**'s ``SIGNING_KEY``;
 * verify that transaction has time bounds set, and that current time is between the minimum and maximum bounds;
-* verify that transaction contains at least one operation;
-* verify that transaction's first operation:
-  * is a Manage Data operation
+* verify that transaction's first operation is a [Manage Data](https://www.stellar.org/developers/guides/concepts/list-of-operations.html#manage-data) operation that:
   * has a non-null source account
-* verify that transaction envelope has a correct signature by server's signing key;
-* if the first operation's source account exists:
-  * verify that client signatures count is one or more;
-  * verify that client signatures on the transaction are signers of the first operation's source account;
-  * verify that client signatures are correct;
-  * verify that client signatures provide weight that meets the required threshold(s), if any;
-* if the first operation's source account does not exist:
-  * verify that client signature count is one;
-  * verify that client signature is correct for the master key of the first operation's source account address;
-* verify that transaction containing additional Manage Data operations have their source account as the server signing key;
+  * has the **Home Domain** as the key of the operation
```

Comment:
💡 It's unnecessary for the server to verify this because they signed the transaction. Our SDKs might do it, but there's little need in the specification to specify it. It doesn't hurt to specify it, but this is why we don't specify verifying the web auth domain here as well.

Reply:
Aren't all verifications above this comment unnecessary though? The only thing the server needs to do is verify that it's account signed the transaction and then verify the other signatures depending on whether the client account exists. Should I remove all verification points above this comment?

Reply:
You're right, but I don't think there is significant value in removing existing statements made here, I just wouldn't go out of your way to add new ones.